### PR TITLE
Adding clarification and a custom CA section to Keystone docs

### PR DESCRIPTION
**Pull request by hyperbolic2346**

### CODE REVIEW

**Comment by evilnick** [comment]
@hyperbolic2346 Sorry, I had about 3 iterations of that, my brain is still warming up - does this look good to you?

**Comment by hyperbolic2346** [comment]
Yes, I like those changes. That sentence was dense and awkward, which is a bad combination. :smile:

File: pages/k8s/ldap.md [hunk 1]
```diff
@@ -197,6 +210,24 @@ juju config kubernetes-master keystone-policy=$(cat policy.yaml)
...
+ * Add CA to client machines that will run `kubectl`.
+
+```
+sudo cp custom_ca.crt /usr/local/share/ca-certificates
+sudo update-ca-certificates
+```
+ * Add CA to the kubernetes-master configuration
+
+```bash
+juju config kubernetes-master keystone-ssl-ca="$(base64 custom_ca.crt)"
```
Comment (Cynerva) [inline pages/k8s/ldap.md]: This config value needs to be a path to the cert, not base64-encoded contents of the cert, right?

Comment (hyperbolic2346) [reply pages/k8s/ldap.md]: @Cynerva not anymore. The path is something mapped into the container, which is impossible for an operator to do without altering the yaml. I have a config.yaml change for the master charm to update the information there to say it is base64 encoded.

Comment (Cynerva) [reply pages/k8s/ldap.md]: Yep, you're right, I misunderstood. Thanks.

**Pull request merged**